block-storage-attacher/tests/e2e/e2e-tests/utilscript.sh

```diff
@@ -25,6 +37,17 @@ then
     export NOD_IP=$3
     bx sl block access-revoke -p $NOD_IP $VOL_ID
     bx sl  block volume-cancel -f --immediate $VOL_ID
+elif [ "$2" = "portworxpvcreate" ]
+then
+    install_portworx_plugin
+    check_portworx_pod_state "portworx"
+    echo "BlockVolumeAttacher-Volume-Test: PortWorx Plugin-Installation: PASS" >> $E2E_PATH/e2eTests.txt
+    export CLSFILE=$3
+    kubectl  create -f $CLSFILE
```

Comment:
You can use `$3` here directly instead of doing `export`.

Reply:
Just for readbility in the script used